.github/workflows/master.yml

```diff
@@ -182,6 +182,14 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: 'maven'
           check-latest: false
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Install a PyFlink dependency
+        run: |
+          python -m pip install --upgrade pip
+          pip install apache-flink==${{ matrix.flink }}.0
```

Comment:
We can get the exact flink version by
```
IT_FLINK=`echo "${{ matrix.flink-archive }}" | grep -E 'flink\-([0-9]+\.[0-9]+.[0-9]+)\-bin' -o | grep -E '[0-9]+\.[0-9]+' -o`
```

Reply:
I found `matrix.flink-archive` may be a '', so I derive a Flink version from `pom.xml` like `mvn help:evaluate -Dexpression=flink.version -Pflink-${{ matrix.flink }} -q -DforceStdout`.